src/llama-arch.cpp

```diff
@@ -46,6 +46,7 @@ static const std::map<llm_arch, const char *> LLM_ARCH_NAMES = {
     { LLM_ARCH_STARCODER2,       "starcoder2"       },
     { LLM_ARCH_MAMBA,            "mamba"            },
     { LLM_ARCH_MAMBA2,           "mamba2"           },
+    { LLM_ARCH_FALCON_H1,        "falcon_h1"        },
```

Comment:
For consistency with the other arch names, use a dash instead of underscore:

```suggestion
    { LLM_ARCH_FALCON_H1,        "falcon-h1"        },
```